tests/Tests.Lambda.LocalRunner/Internal/RunnerBuilderTests.cs

```diff
@@ -17,11 +17,13 @@ public void Receives_T_returns_initialized_builder_step(InnerRunnerBuilder sut,
         {
             sut.UsePort(port);
             sut.UseSerializer(serializerFactory);
+            sut.WithResponseContentType("application/xml");
```

Comment:
Since there is no specific logic validating the content type, you can accept the content type as an argument of the test.

Reply:
Fixed